Sources/Paywalls/Components/PaywallPurchaseButtonComponent.swift

```diff
@@ -14,17 +14,41 @@ public extension PaywallComponent {
 
     struct PurchaseButtonComponent: PaywallComponentBase {
 
-        let type: String
-        let cta: PaywallComponent.LocalizationKey
-        let ctaIntroOffer: PaywallComponent.LocalizationKey?
+        let type: ComponentType
+        public let ctaLid: LocalizationKey
+        public let ctaIntroOfferLid: LocalizationKey
```

Comment:
Just a suggestion, not a blocker by any means, but imo these don't _need_ to have the `*Lid` suffix. The fact that it is typed as `LocalizationKey` already tells me it's a localization thing. 

```diff
-        public let ctaLid: LocalizationKey
-        public let ctaIntroOfferLid: LocalizationKey
+        public let cta: LocalizationKey
+        public let ctaIntroOffer: LocalizationKey
```

Reply:
I _do_ agree that here it is fine but I think we will still want to have `_lid` in the JSON otherwise its super unclear 🤷‍♂️ 

@JayShortway  We can make this change and then just map `cta_lid` to `cta` in this object 💪 Thoughts?

Reply:
Oh right, I'm still not used to Swift's magic JSON (de)serialization. In that case I actually don't mind as much haha. Might be good to directly map the JSON with this object. (I agree that we need `_lid` in the JSON.)